examples/advanced/edge/README.md

```diff
@@ -0,0 +1,66 @@
+# Running Edge Example with Hierarchical Clients
+
+Please follow these steps to run the Edge device emulator,
+
+## Provision
+
+Use tree_prov.py to generate a hierarchical NVFlare system with 2 levels and 2 clients at each level,
+
+     python tree_prov.py -p /tmp/edge_example -d 2 -w 2
+
+This will create a deployment with 2 clients, 4 leaf-clients, 2 relays, 1 server.
+
+This file needs to be copied to the `local` folder of each leaf clients, C11, C12, C21 and C22.
+
+`edge__p_resources.json`:
+
+```
+{
+  "format_version": 2,
+  "components": [
+    {
+      "id": "web_agent",
+      "path": "nvflare.edge.widgets.web_agent.WebAgent",
+      "args": {}
+    },
+    {
+      "id": "etd",
+      "path": "nvflare.edge.widgets.etd.EdgeTaskDispatcher",
+      "args": {}
+    }
+  ]
+}
+```
+
+To start the system, just run the following command,
+    ./start_all.sh
```

Comment:
within the prepared workspace folder /tmp/edge_example/prod_00/

Reply:
Changed.